app/ai.py

```diff
@@ -88,18 +89,32 @@ def yolo_forward(net, LABELS, image, confidence_level, save_image=False):
 
     # apply non-maxima suppression to suppress weak, overlapping bounding
     # boxes
-    # idxs = cv2.dnn.NMSBoxes(boxes, confidences, confidence_level, threshold)
-
+    idxs = cv2.dnn.NMSBoxes(boxes, confidences, confidence_level, confidence_level)
+    
+    
     print(class_ids)
-    print(LABELS)
-    # print(labels)
+    print(boxes)
+    print(confidences)
+    print(idxs)
+    
+    if len(idxs) > 0:
+        filtered_idxs = idxs[0]
```

Comment:
the verison I have has this instead

 if len(idxs) > 0:
        # Michael's version - this one only provides 1 label per image.
        # filtered_idxs = idxs[0]
        # Pierre Quereuil change on June 29 , 2021 to allow for multiple labels
  ==>      filtered_idxs =  np.concatenate(idxs)
        print('after NMS, we have these indices')
        print(filtered_idxs)

Reply:
fixed by adding Pierre's modification